jena-arq/src/main/java/org/apache/jena/riot/writer/DirectiveStyle.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.jena.riot.writer;
+
+enum DirectiveStyle {
+    AT, SPARQL ;
+    public static DirectiveStyle create(String label) {
+        String s = label.toLowerCase() ;
+        switch(s) {
+            case "rdf_10": case "rdf10": case "n3": case "at":
+                return DirectiveStyle.AT ;
+            case "rdf_11": case "rdf11": case "sparql":
+                return DirectiveStyle.SPARQL ;
+        }
+        return null;
+    }
+}
```

Comment:
Missing `newline` at the end of the file? Important for git I think...

Reply:
OK - done.